azurerm/resource_arm_cognitive_account_test.go

```diff
@@ -0,0 +1,115 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	// "github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+// TODO: an update test going from External -> Internal -> External
+
+func TestAccAzureRMCognitiveAccount_basic(t *testing.T) {
```

Comment:
can we add a secondary `TestAccAzureRMCognitiveAccount_complete` test featuring some Tags?

Reply:
can we also add an `_import` test for both of these in `azurerm/import_arm_cognitive_account_test.go`? there's an example of what this looks like here: https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/import_arm_key_vault_test.go